apps/web/pages/_document.tsx

```diff
@@ -1,58 +1,36 @@
 import Document, { DocumentContext, Head, Html, Main, NextScript, DocumentProps } from "next/document";
 import Script from "next/script";
+import { z } from "zod";
 
 import { getDirFromLang } from "@calcom/lib/i18n";
 
-type Props = Record<string, unknown> & DocumentProps;
-
-function toRunBeforeReactOnClient() {
```

Comment:
Moved this to embed-init-iframe.ts so that inline code can be avoided.
Note that: This might not be necessary with strict-dynamic. But if we switch to non strict CSP policy it would come in handy.